fin_control/src/fin_control.cpp

```diff
@@ -57,7 +57,7 @@ FinControl::FinControl(ros::NodeHandle &nodeHandle)
   fincontrolEnabled = false;
   std::string serial_dev = "/dev/ttyUSB0";
   int serial_baud = 57600;
-  maxCtrlFinAngle = 10.0;
+  maxCtrlFinAngle = 0.174533;  //  10.0 degree
```

Comment:
@patapufate nit: I'd suggest 
```suggestion
  constexpr double degrees = M_PI / 180.0;
  maxCtrlFinAngle = 10 * degrees;
```
instead.

Reply:
The autopilot publishes the fins c ontrol in radians in radians.

Reply:
Changed